rust/routee-compass/src/app/compass/response/write_mode.rs

```diff
@@ -24,6 +28,7 @@ impl WriteMode {
                 open_append(path)
             }
             WriteMode::Overwrite => {
+                remove_if_exists(path)?;
```

Comment:
this action was missing from a truly "overwrite" workflow where it should first remove any file at the output path if it exists.